components/pd_client/src/client.rs

```diff
@@ -530,6 +530,7 @@ impl PdClient for RpcClient {
         req.set_query_stats(region_stat.query_stats);
         req.set_approximate_size(region_stat.approximate_size);
         req.set_approximate_keys(region_stat.approximate_keys);
+        req.set_cpu_time_ms(region_stat.cpu_time_ms);
```

Comment:
I wonder the unit `cpu time ms` is different with the unit of the store heartbeat `CpuUsages`. better to unify them.

Reply:
Done. PTAL.